packages/@react-aria/menu/stories/useMenuTrigger.stories.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {DismissButton, useOverlay} from '@react-aria/overlays';
+import {Flex} from '@react-spectrum/layout';
+import {FocusScope} from '@react-aria/focus';
+import {Item} from '@react-stately/collections';
+import {mergeProps} from '@react-aria/utils';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {useButton} from '@react-aria/button';
+import {useFocus} from '@react-aria/interactions';
+import {useMenu, useMenuItem, useMenuTrigger} from '@react-aria/menu';
+import {useMenuTriggerState} from '@react-stately/menu';
+import {useTreeState} from '@react-stately/tree';
+
+
+storiesOf('useMenuTrigger', module)
+  .add('2 menus', () => (
+    <Flex>
+      <MenuButton label="Actions">
+        <Item key="copy">Copy</Item>
+        <Item key="cut">Cut</Item>
+        <Item key="paste">Paste</Item>
+      </MenuButton>
+      <MenuButton label="Actions2">
+        <Item key="copy">Copy</Item>
+        <Item key="cut">Cut</Item>
+        <Item key="paste">Paste</Item>
+      </MenuButton>
+    </Flex>
+  ))
+  .add('2 menus with disabled', () => (
+    <Flex>
+      <MenuButton isDisabled label="Actions">
+        <Item key="copy">Copy</Item>
+        <Item key="cut">Cut</Item>
+        <Item key="paste">Paste</Item>
+      </MenuButton>
+      <MenuButton label="Actions2">
+        <Item key="copy">Copy</Item>
+        <Item key="cut">Cut</Item>
+        <Item key="paste">Paste</Item>
+      </MenuButton>
+    </Flex>
+  ));
+
+function MenuButton(props) {
+  // Create state based on the incoming props
+  let state = useMenuTriggerState(props);
+
+  // Get props for the menu trigger and menu elements
+  let ref = React.useRef(null);
+
+  let shouldCloseOnInteractOutside = (element) => !ref?.current?.contains(element) ?? false;
```

Comment:
Why is this prop needed? Would be nice to avoid making everyone writing an overlay pass it in...